### PR TITLE
Handling of unknown categories

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

File: extension-manifest-v3/src/pages/panel/views/detailed.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default define({
       ${store.ready(stats) &&
       html`
         ${Object.keys(stats.byCategory)
-          .sort((a, b) => order.indexOf(a[0]) - order.indexOf(b[0]))
```
Comment (chrmod) [inline extension-manifest-v3/src/pages/panel/views/detailed.js:31]: looks like we had a bug in here

File: ui/src/utils/categories.js [hunk 2]
```diff
@@ -89,3 +94,24 @@ export const labels = {
...
+
+export const getCategoryColor = (category) => {
+  return colors[getCategoryKey(category)];
+};
+
+export const getCategoryLabel = (category) => {
+  return labels[getCategoryKey(category)];
+};
+
+export const sortCategories = (_resolveCategoryName) => {
```
Comment (smalluban) [inline ui/src/utils/categories.js:110]: You can avoid underscore prefix and just do:
```suggestion
export const sortCategories = (resolveCategoryName = ((a) => a)) => {
```

**Review by smalluban — CHANGES_REQUESTED** [review]
Categories are shared between v2 and v3, so you should also run `npm run locales` in v2, to add new category to its localization files.

File: extension-manifest-v3/src/pages/panel/views/detailed.js (hunk 1 above)
Comment (smalluban) [reply extension-manifest-v3/src/pages/panel/views/detailed.js:31]: I assume that added `sortCategories` function solves the problem, isn't it? ;)

**Review by smalluban — APPROVED** [review]
It's fine now, but I made a typo in the suggestion, so there are to many braces, you must remove one.

**Pull request merged**